edge-functions/edge-config-statsig/README.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Experimentation with Statsig
+slug: edge-config-statsig
+description: By running experiments at the edge with Statsig, you'll reduce CLS and improve performance from client-loaded experiments while analyzing impact.
+framework: Next.js
+useCase:
+  - Edge Functions
+  - Documentation
+css: Tailwind
+deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/edge-config-statsig&project-name=edge-config-statsig&repository-name=edge-config-statsig
```

Comment:
Is this missing the required envs? The link on the Deploy button further down has them